.github/workflows/ci.yml

```diff
@@ -7,6 +7,7 @@ on: [push, pull_request]
 jobs:
   tests:
     runs-on: ${{ matrix.os }}
+    if: github.event.pull_request.draft == false
```

Comment:
I have mixed feelings about this. Let's put it in a different PR and discuss there

Reply:
It's wrong: I meant `!=` false, i.e. it should only run tests on non-draft PRs.

Reply:
Sure.